### DISCUSSION TITLE
Yay badges!

### DISCUSSION THREAD

**Discussion by Nquenan** [doctorixx/Getting_badges#7] [Q&A] [answered]
Badges?

**Comment by aniruddha76** [comment, marked as answer]
trying to get galaxy brain badge too, mark this as answer : )

**Reply by doctorixx** [reply]
: )

**Reply by aniruddha76** [reply]
thanks : )

**Comment by anups1ngh** [comment]
Mark me as well please !

**Reply by doctorixx** [reply]
I can mark only one comment as answer.
Sorry)

But you can create a new discussion
